packages/edit-navigation/src/components/menus-editor/index.js

```diff
@@ -24,28 +37,60 @@ export default function MenusEditor( { blockEditorSettings } ) {
 		}
 	}, [ menus ] );
 
-	if ( ! stateMenus ) {
+	if ( ! hasLoadedMenus ) {
 		return <Spinner />;
 	}
 
+	const hasMenus = hasLoadedMenus && !! stateMenus?.length;
+	const isCreatingFirstMenu = ! hasMenus;
+	const isCreatingAdditionalMenu = hasMenus && ! menuId;
+	const isCreatingMenu = isCreatingFirstMenu || isCreatingAdditionalMenu;
+	const hasSelectedMenu = hasMenus && !! menuId;
+
 	return (
 		<>
 			<Card className="edit-navigation-menus-editor__menu-selection-card">
-				<CardBody>
-					<SelectControl
-						className="edit-navigation-menus-editor__menu-select-control"
-						label={ __( 'Select navigation to edit:' ) }
-						options={ stateMenus.map( ( menu ) => ( {
-							value: menu.id,
-							label: menu.name,
-						} ) ) }
-						onChange={ ( selectedMenuId ) =>
-							setMenuId( selectedMenuId )
-						}
-					/>
+				<CardBody className="edit-navigation-menus-editor__menu-selection-card-body">
+					{ isCreatingMenu && (
+						<p className="edit-navigation-menus-editor__menu-selection-card-instructional-text">
+							{ isCreatingFirstMenu
+								? __( 'Create your first menu below.' )
+								: __( 'Create a new menu below.' ) }
+						</p>
+					) }
+					{ hasSelectedMenu && (
+						<>
+							<SelectControl
+								className="edit-navigation-menus-editor__menu-select-control"
+								label={ __( 'Select navigation to edit:' ) }
+								options={ stateMenus?.map( ( menu ) => ( {
+									value: menu.id,
+									label: menu.name,
+								} ) ) }
+								onChange={ ( selectedMenuId ) =>
+									setMenuId( selectedMenuId )
+								}
+							/>
+							<Button isLink onClick={ () => setMenuId() }>
```

Comment:
Not a huge fan of setting the menu id to `undefined` to indicate a menu is being created. Open to suggestions for improving this. Potentially could use some state, e.g.
```
editMode: 'CREATE_MENU'
```